### ISSUE TITLE
[tabs] Keyboard navigation is incomplete, broken in chrome - DDS Consulting

### ISSUE THREAD

**Issue by andy-blum** [carbon-design-system/carbon-web-components#707]
###  Engineering info:
- HC engineer: 
- HC Jira Ticket: 
- DDS consulting engineer: 

----

## Detailed description

> Describe in detail the issue you're having.

Tabbed interfaces using `role="tablist"` and `role="tabitem"` carry an implicit interaction model that needs to be followed for WCAG compliance. Right now, that UI pattern is incomplete and broken in Chrome.

> Is this a feature request (new component, new icon), a bug, or a general
> issue?

:bug: Bug

> Is this issue related to a specific component?

Tabs

> What did you expect to happen? What happened instead? What would you like to
> see changed?

1. Missing keyboard interactions:
- While focus is within the tablist, the `Home` key should move focus to the **first** tab and activate it.
- While focus is within the tablist, the `End` key should move focus to the **last** tab and activate it.
- While focus is within the tablist, the `Tab` key should move focus to the active tabpanel element, and then to any focusable elements within that tabpanel before moving on to other parts of the page

2. Chrome's keyboard interaction model is completely broken at the moment. For some reason the links within each `<bx-tab>` are unable to receive/hold focus.

> What browser are you working in?

Tested in the following:
- Chrome 94.0.4606.61
- Firefox 92.0.1
- Safari 15.0 (16612.1.29.41.4, 16612)

> What version of Carbon Web Components are you using?

Latest at time of writing ([#01b2892](https://github.com/carbon-design-system/carbon-web-components/commit/01b289238cd9a8edceaa3ad1ee548cb6693a5da8))

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

carbon-for-ibm-dotcom, but we've already fixed this issue [downstream](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7188) for now.

## Additional information

- [W3: Tabs example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html#kbd_label)

**Comment by RobertaJHahn** [comment]
@Praveen-sr Could you have Ram take a look to verify this bug and set the severity.  Thanks.

**Comment by RobertaJHahn** [comment]
Praveen and Ram have started the testing.

**Comment by andy-blum** [comment]
@annawen1 if `tabs-extended` isn't inheriting from this repo's `tabs` then this issue is only related in the sense that they should have similar solutions.

**Comment by annawen1** [comment]
@andy-blum I wonder if we should keep separate tickets or create one ticket to add solution to both `carbon-for-ibm-dotcom` and `carbon-web-component` tab component

**Comment by RobertaJHahn** [comment]
@annawen1 @andy-blum I think we have 2 separate issues.  #5816 is for Carbon for ibm.com and this issue is for Carbon web components.  I have added the severity to this issue.  I think we can move it forward into the Pending scheduling pipeline.

**Comment by kennylam** [comment]
Duplicate of #970. Capturing remaining a11y requirements (`home`/`end`) there.
